android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerFragment.kt

```diff
@@ -0,0 +1,36 @@
+package com.brentvatne.exoplayer
+
+import android.os.Build
+import androidx.fragment.app.Fragment
+
+class ReactExoplayerFragment(private val view: ReactExoplayerView) : Fragment() {
+
+    private var mIsOnStopCalled = false
+
+    override fun onStart() {
+        super.onStart()
+        mIsOnStopCalled = false
+    }
+
+    override fun onStop() {
+        // On entering Picture-in-Picture mode, onPause is called, but not onStop.
+        // For this reason, this is the place where we should pause the video playback.
+        super.onStop()
+        if (!view.playInBackground) view.setPausedModifier(true)
+        mIsOnStopCalled = true
+
+        // Handling when onStop is called while in multi-window mode
+        if (Build.VERSION.SDK_INT >= 24 && activity?.isInMultiWindowMode == true && activity?.isInPictureInPictureMode != true && !view.playInBackground) {
```

Comment:
Use predefined version name
```suggestion
        // Handling when onStop is called while in multi-window mode
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N && activity?.isInMultiWindowMode == true && activity?.isInPictureInPictureMode != true && !view.playInBackground) {
```

Reply:
resolve with [8cf55f3](https://github.com/TheWidlarzGroup/react-native-video/pull/3385/commits/8cf55f3e935815a318143ccd459448f235055d13)